include/dbclean.php

```diff
@@ -16,22 +16,48 @@
 load_config('config');
 load_config('system');
 
-update_shadow_copy();
+remove_orphans();
```

Comment:
[standards] As per PSR-1 coding standards, a file should either declare symbols (variables, functions, classes) or perform actions, but not both.

Ideally, the remove_orphans() function would be declared in a different library or class file.